src/collection.ts

```diff
@@ -537,30 +537,30 @@ export abstract class BaseCollection implements C8Collection {
     );
   }
 
-  createFulltextIndex(fields: string[] | string, minLength?: number) {
+  createFulltextIndex(fields: string[] | string, opts?: any) {
     if (typeof fields === "string") {
       fields = [fields];
     }
     return this._connection.request(
       {
         method: "POST",
         path: "/index",
-        body: { fields, minLength, type: "fulltext" },
+        body: { ...opts, fields, type: "fulltext" },
         qs: { collection: this.name },
       },
       (res) => res.body
     );
   }
 
-  createTtlIndex(fields: string[] | string, expireAfter: number) {
+  createTtlIndex(fields: string[] | string, expireAfter: number, name?: string) {
```

Comment:
jsc8 has one of the more "maintained" test cases. We should try to maintain them whereever possible. As CLI also depends on jsc8, it becomes necessary to a certain extent that we are confident on jsc8

Reply:
@lohaniabhishek Sure we will do that. Do you want us to include in the master and then publish the beta?

Reply:
yes please include test if possible